src/component/contents/DraftEditorBlock.react.js

```diff
@@ -204,6 +213,7 @@ class DraftEditorBlock extends React.Component {
     return (
       <div data-offset-key={offsetKey} className={className}>
         {this._renderChildren()}
```

Comment:
We still need `_renderChildren()` to render the inner text (with styles and entities).

`_renderBlockMap()` can only render the children blocks.

I think we should keep it this way: block should still have `text`, `styleRanges` and `entitieRanges`, even when they have a `blockMap`, it will ensure compatibility with previous versions of Draft (only extend it).